src/schema/home/home_page_artwork_module.ts

```diff
@@ -92,10 +92,16 @@ const HomePageArtworkModule: GraphQLFieldConfig<void, ResolverContext> = {
     switch (key) {
       case "generic_gene":
         return { key, display, params: find(genericGenes, ["id", id]) }
+      // Emission may include an `id` param here,
+      // but Force does not.
       case "genes":
-        return geneLoader(id).then(gene => {
-          return { key, display, params: { id, gene } }
-        })
+        if (id) {
```

Comment:
Related, I have a Force PR that will not pass the string `"undefined"`, but the value `undefined`, which makes this check work.